client/src/constants/layout.ts

```diff
@@ -0,0 +1 @@
+export const LAYOUT_BODY_ID = 'LayoutBody';
```

Comment:
I think it would be better to put it into `metrics.ts` file and rename to `domElementsIds.ts`, so it's easier to find. IDs, as an important attribute of DOM elements that shouldn't be repeated, better keep in one place.

What do you think?

Reply:
@aziolek  Good idea. I changed it ;)